### PR TITLE
feature(backend): add a new column of categories

### LINKED ISSUE #585: Offer approval  create categories
- [x] Add category column in DB

### PR DESCRIPTION
**Pull request by Edgar097**
Resolves #585 

### What does this PR do?
Backend logic to add categories to the lifebank system

### How should this be tested?
- Uncomment Front-End Connection Example at line 144, path "lifebank/webapp/src/components/Signup/SignupLifeBank.js"
-  Make run
-  Execute :add a new lifebank use case

### CODE REVIEW

**Review by steph222 — APPROVED** [review]

File: webapp/src/components/Signup/SignupLifeBank.js [hunk 1]
```diff
@@ -140,6 +140,18 @@ const SignupLifeBank = ({
...
+          /*
+          <TextField
+            id="name"
+            label={'categories'}
+            variant="outlined"
+            fullWidth
+            className={classes.textField}
+            onChange={(event) => setField('categories', ["Coupons", "Discount", "Gift"])}
+          /> */
+        }
```
Comment (leisterfrancisco) [inline webapp/src/components/Signup/SignupLifeBank.js:154]: Check this comments

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

File: webapp/src/components/Signup/SignupLifeBank.js (hunk 1 above)
Comment (Edgar097) [reply webapp/src/components/Signup/SignupLifeBank.js:154]: It's made on purpose if you want to test it, if everything is ok i will commit the removal of those lines in order to merge

Comment (leisterfrancisco) [reply webapp/src/components/Signup/SignupLifeBank.js:154]: I supposed that but I think we could remove those lines if we want to merge it to master

**Review by leisterfrancisco — APPROVED** [review]

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**